### ISSUE TITLE
Configure Artstor Shared Shelf data in Aggregator

### ISSUE THREAD

**Issue by dorevabelfiore-temple** [tulibraries/dplah#132]
We will need to configure the aggregator to work with Artstor Shared Shelf as harvested from Artstor. Currently in Being Prepped waiting for new IP address of new test server, to share with Artstor.

**Comment by EliseTemple** [comment]
Doreva & Leanne will assess the XML first and then schedule a meeting with Chad once we have some test data.

**Comment by dorevabelfiore-temple** [comment]
Still working with PMA to get test data.

**Comment by dorevabelfiore-temple** [comment]
Waiting on PMA who is consulting with their lawyers & administration.

**Comment by dorevabelfiore-temple** [comment]
PMA is on hold indefinitely and will contact us.
